### DISCUSSION TITLE
E3V2-UBL-BLTouch-10x10-v4.2.2 probes only 63 points

### DISCUSSION THREAD

**Discussion by PapsDizzy** [Jyers/Marlin#813] [Q&A] [answered]
Hi there,
is this normal that under creat new mesh only 63 points will probe and then saved?
Sorry for the questin im new in ubl and bltouch

**Comment by smooreace** [comment]
short answer...Yes

**Reply by PapsDizzy** [reply]
think im right that the rest of points is calculatet?
or why is only 63 points of 100 been probed?

**Comment by smooreace** [comment]
Inferred is the more commonly used word.. The rest of the points
arent reachable because of the probe offset. If your probe is to the left
of the nozzle then it can't get to the far right of the print surface. If
it's on the other side then same thing on the left

**Comment by smooreace** [comment, marked as answer]
So a 10x10 mesh means a probe every 23mm based on a 230x230 bed. Take off
your probe offset from the right side of the bed (assuming your probe is
left of the nozzle) and you get an approximate far right reach of the
probe. Same thing actually for the Y direction.... probe offset make it
physically impossible to got to parts of the bed
